CMakeLists.txt

```diff
@@ -564,6 +565,37 @@ endif()
 
 if(NOT IREE_BUILD_COMPILER)
   message(STATUS "Not adding LLVM/MLIR because the configuration does not require it")
+elseif(NOT IREE_BUILD_BUNDLED_LLVM)
+  message(STATUS "Using installed LLVM components")
+  find_package(LLVM REQUIRED)
+  find_package(MLIR REQUIRED)
+  find_package(LLD REQUIRED)
+  list(APPEND CMAKE_MODULE_PATH "${LLVM_DIR}")
+  list(APPEND CMAKE_MODULE_PATH "${MLIR_DIR}")
+  list(APPEND CMAKE_MODULE_PATH "${LLD_DIR}")
+  include(TableGen)  # TODO: Eliminate.
+  message(STATUS "Installed LLVM packages found")
+  message(STATUS "LLVM_INCLUDE_DIRS=${LLVM_INCLUDE_DIRS}")
+  #message(STATUS "LLVM_LIBRARY_DIR=${LLVM_LIBRARY_DIR}")
```

Comment:
these commented lines needed? we have a lot of spam, may as well have two more if useful?